### ISSUE TITLE
Set Connected Service for a user by API

### ISSUE THREAD

**Issue by klatka** [goauthentik/authentik#1888]
**Is your feature request related to a problem? Please describe.**
I am migrating users and want to add their google uuid to the account I created for them without having them enrol it themselves.

**Describe the solution you'd like**
An endpoint like `POST /sources/user_connections/oauth/`
```
REQUEST BODY application/json
{
    source*: uuid
    identifier*: string
}
```
Get `userId` from Cookie or API Key.

Response should be like `GET /sources/user_connections/oauth/`

**Comment by klatka** [comment]
I updated to 2021.12.1. POST /sources/user_connections/oauth/ gives me 405 Method Not Allowed

**Comment by BeryJu** [comment]
My bad, I forgot to make the user field on that endpoint writeable, this will be fixed in 2021.12.2
